content/_partials/downloadbanner.html.haml

```diff
@@ -0,0 +1,58 @@
+:javascript
+  $(function () {
+    function doPopovers(){
+      var $popovers = $('[data-toggle="popover"]');
+      var $popLayer = $('#popover-layer');
+      if ($popovers.length > 0){
+        if($popLayer.length < 1) 
+          $popLayer = $('<div/>').appendTo('body').addClass('popover-layer').attr('id','popover-layer');
+        $('[data-toggle="popover"]').popover({
+          container: '#popover-layer',
+          template:'<div class="popover download-options" role="tooltip"><div class="popover-arrow"></div><div class="popover-content"></div></div>',
+          placement:'top',
+          content:function(){
+            return $(this).next().html();
+          },
+          html:true
+        });
+      }
+    }
+    doPopovers();
+  })
+  
+%div#ji-home-carousel.carousel.slide{'data-ride' => 'carousel',:class => page.topic}
+  #ji-download
+    .btn-group
+      %button.btn-lg.btn.btn-primary{'data-toggle'=>"popover", 'aria-haspopup'=>"true", 'aria-expanded'=>"false"}
+        Download Jenkins
+      = partial('downloadlist.html.haml')
+    .br
+      .carousel-caption.secondary
+        Get 1.625.3 LTS .war or the latest 1.644 weekly or choose other options
+  %ol.carousel-indicators.hidden
+    %li.active{'data-target' => '#ji-home-carousel', 'data-slide-to' => 0}
+    %li{'data-target' => '#ji-home-carousel', 'data-slide-to' => 1}
+    %li{'data-target' => '#ji-home-carousel', 'data-slide-to' => 2}
+    %li{'data-target' => '#ji-home-carousel', 'data-slide-to' => 3}
+    %li{'data-target' => '#ji-home-carousel', 'data-slide-to' => 4}
+  %div.carousel-inner{:role=>"listbox"}
+    %div.carousel-item.active
+      %video{:autoplay => ""}
+        \//EDIT: loop="true" and  autoplay="autoplay" are deprecated
+        %source{:src => "/images/grow-city-sketch.mp4", :type => "video/mp4"}
+        Your browser does not support the video tag.    
+      .mask
+      %div.carousel-caption.primary
+        %img.jenkins{:src=>"/images/226px-Jenkins_logo.svg.png"}
```

Comment:
The video may well be too heavy and may not be the right aesthetic. At the moment, I am tossing stuff into the soup and listening for comments.